.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+<!--
+  Thanks for creating a Pull Request! Before you submit, please make sure
+  you've done the following:
+
+  - I read the contributing document at https://jekyllrb.com/docs/contributing/
+-->
+
+<!--
+  Make our lives easier! Choose one of the following by uncommenting it:
+-->
+
+<!-- This is a 🐛 bug fix. -->
+<!-- This is a 🙋 feature or enhancement. -->
+<!-- This is a 🔦 documentation change. -->
+
+<!--
+  Before you submit this pull request, make sure to have a look at the following
+  checklist. If you don't know how to do some of these, that's fine! Submit
+  your pull request and we will help you out on the way.
+
+  - I've added tests (if it's a bug, feature or enhancement)
+  - I've adjusted the documentation (if it's a feature or enhancement)
+  - The test suite passes (run `script/cibuild` to verify this)
```

Comment:
Should these be actual *checkboxes* when uncommented..? That's not clear..
e.g:

  - [x] I've added tests

Reply:
no, i precisely didn't add checkboxes because the syntax is confusing, meaning most people don't actually check them

Reply:
this is just a checklist that only exists in the comment